### DISCUSSION TITLE
Can someone link me a modchart guide?

### DISCUSSION THREAD

**Discussion by ZenusPurity** [KadeArchive/Kade-Engine#505] [Q&A]
Can someone link me a modchart guide? With all the commands and such

**Comment by ZenusPurity** [comment]
Also, can modcharts be difficulty-specific? 

There's a modchart on hard but not on normal, is that possible?

**Reply by isakube** [reply]
Nope. also modchart guide https://kadedev.github.io/Kade-Engine/modchart
